subsystems/provisioning/provisioning.go

```diff
@@ -88,6 +103,9 @@ func NewProvisioning(ctx context.Context, logger logging.Logger, updateConf *age
 
 		mainLoopHealth: &health{},
 		bgLoopHealth:   &health{},
+
+		muProvisioningMethod: sync.Mutex{},
+		provisioningMethod:   provisioningMethodBLE,
```

Comment:
FYI i'll confirm with Eliot which one should be the default. My instinct says BLE but